demo/start.sh

```diff
@@ -1,24 +1,29 @@
 #!/usr/bin/env bash
 
+# Disable unicode.
+LC_ALL=C
+LANG=C
+
 DIR="$( cd "$( dirname "$0" )" && pwd )"
 
 CWD=$(pwd)
 
-cd "$DIR/.."
+cd "$DIR/.." || exit
 
 mkdir -p demo/etc
 
-export PYTHONPATH=$(readlink -f $(pwd))
+export PYTHONPATH
+PYTHONPATH=$(readlink -f "$(pwd)")
```

Comment:
these two are equivalent, as far as I can tell.

Reply:
Is good to declare and assign separately to avoid masking return values
# References
[https://github.com/koalaman/shellcheck/wiki/SC2155](https://github.com/koalaman/shellcheck/wiki/SC2155)

Reply:
TIL, thanks!